### PR TITLE
PLAT-10414: Upgraded dependencies

### PR DESCRIPTION
**Pull request by symphony-elias**
### Ticket
PLAT-10414

### Description
Upgraded dependencies and node version to latest LTS version.
Set up CircleCI.

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [x] Unit tests updated or added
- [N/A] Public functions properly commented

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1.0
+
+workflows:
+  node-tests:
+    jobs:
+      - node/test:
+          version: 14.15.4
```
Comment (symphony-youri) [inline .circleci/config.yml:10]: that's one simple job! too bad it does not support publishing the test reports out of the box

File: lib/ConnectionsClient/index.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ ConnectionsClient.getRejectedConnections = (sessionToken) =>
 ConnectionsClient.getConnections = (status, userIds, sessionToken) =>
   podRequest(
     'GET',
-    `/pod/v1/connection/list?status=all?userIds=${userIds}`,
+    `/pod/v1/connection/list?status=all&userIds=${userIds}`,
```
Comment (symphony-youri) [inline lib/ConnectionsClient/index.js:65]: How did you catch that ? :)

**Review by symphony-youri — COMMENTED** [review]
@symphony-thibault If you are admin on the repo could you enable the automatic linking to Jira?

**Review by symphony-youri — APPROVED** [review]

**Comment by symphony-thibault** [comment]
@symphony-elias Could we take the opportunity to fix some of the [opened issues](https://github.com/SymphonyPlatformSolutions/symphony-api-client-node/issues)? If possible and not too long obviously

**Comment by symphony-elias** [comment]
> @symphony-elias Could we take the opportunity to fix some of the [opened issues](https://github.com/SymphonyPlatformSolutions/symphony-api-client-node/issues)? If possible and not too long obviously

Closed several issues. Will try to fix #59 if possible.

**Pull request merged**